src/rust/engine/fs/src/store.rs

```diff
@@ -982,7 +975,12 @@ mod local {
             Err(err) => Err(format!("Error storing digest {:?}: {}", digest, err)),
           }
         })
-        .to_boxed()
+      })
+      .then(|blocking_result| match blocking_result {
+        Ok(v) => v,
+        Err(blocking_err) => panic!("TODO: {}", blocking_err),
```

Comment:
TODO

Reply:
Oops! Done. There's not an obviously good thing to do here; we could either directly panic, or return a String error... Switched to return an error.